File: Inventory_backend/Entities/Concrete/Order.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Entities.Concrete
{
    public class Order : IEntity
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public Customer Customer { get; set; }
        public int ProductId { get; set; }
        public Product Product { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public decimal Total { get; set; }
    }
}

File: Inventory_frontend/src/app/models/order.ts
import { Customer } from "./customer";
import { Product } from "./product";

export class Order{
    id:number;
    productId:number;
    product:Product;
    customerId:number;
    customer:Customer;
    price:number;
    quantity:number;
    total:number;
    constructor(productId:number,customerId:number,price:number,quantity:number,total:number){
     this.productId=productId
     this.customerId=customerId
     this.price=price
     this.quantity=quantity
     this.total=total
    }
}